CMakeLists.txt

```diff
@@ -544,14 +544,12 @@ IF (CXXTEST_FOUND)
 		)
 	ENDIF (CMAKE_BUILD_TYPE STREQUAL "Coverage")
 	ADD_CUSTOM_TARGET(test_python
-		DEPENDS PythonEvalUTest
-				PythonUtilitiesUTest
-				bindlink_cython atomspace_cython
-				scheme_wrapper logger_cython utilities_cython
+		DEPENDS PythonEval logger_cython scheme_wrapper bindlink_cython atomspace_cython
```

Comment:
PythonEvalUTest and PythonUtilitiesUTest are still missing.  However, PythonEval is not a unit test, so it cannot actually run. 

Did you actually try to run `make test_python` ?  when you say that, it should run all (seven) python tests.

Reply:
Oh, I see ... OK, dependencies. Yes, I guess these are sqonky.  Never mind; I'll just merge this.